bundles/framework/layerlist/view/LayerViewTabs/LayerList/LayerCollapse/LayerCollapseHandler.js

```diff
@@ -89,7 +90,7 @@ class ViewHandler extends StateHandler {
         // For non admin users empty arrays are provided and with this empty groups are not included to layerlist.
         const allGroups = isUserAdmin ? this.mapLayerService.getAllLayerGroups() : [];
         const allDataProviders = isUserAdmin ? this.mapLayerService.getDataProviders() : [];
-        let groups = groupLayers([...layers], this.groupingMethod, tools, allGroups, allDataProviders);
+        let groups = groupLayers([...layers], this.groupingMethod, tools, allGroups, allDataProviders, this.loc);
```

Comment:
Maybe just pass the loc.grouping.noGroup here so the util method becomes more testable?